server/ctrlsubsonic/handlers_playlist.go

```diff
@@ -90,10 +93,17 @@ func (c *Controller) ServeCreatePlaylist(r *http.Request) *spec.Response {
 		FirstOrCreate(&playlist)
 
 		// update meta info
-	playlist.UserID = user.ID
+	if playlist.UserID == 0 {
+		playlist.UserID = user.ID
+	} else {
+		if playlist.UserID != user.ID {
+			return spec.NewResponse()
+		}
+	}
 	if val, err := params.Get("name"); err == nil {
 		playlist.Name = val
 	}
+	playlist.IsPublic = false
```

Comment:
in go, the zero value of bools is "false" (and you can't have a nil bool)

so we could say, if we like:

```suggestion
```